experiment_pages/experiment/experiment_menu_ui.py

```diff
@@ -101,7 +104,10 @@ def delete_experiment(self, page: CTkFrame, name: str):
         splitted = name.split("\\")
         if ("Protected" in splitted[-1]):
             path = os.getcwd()
-            name = path + "\\databases\\experiments\\" + splitted[-1]
+            if os.name == 'posix':
```

Comment:
This code looks good to me, simply checks out if we are on Windows or MacOS. What would happen if this ran on Linux?